lerna.json

```diff
@@ -11,7 +11,6 @@
     "filesystem",
     "geolocation",
     "haptics",
-    "keyboard",
```

Comment:
Should we remove everything but what we've modified?

Reply:
Yes and it worked.